src/components/PopupWithDeleteCard.js

```diff
@@ -0,0 +1,38 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDeleteCard extends Popup {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.

Reply:
Thanks, I like how I can finally listen and respond to a reviewer.